File: 1672/main.go
package main

import (
	"fmt"
)

func maximumWealth(accounts [][]int) int {
	var res int = 0

	for i := 0; i < len(accounts); i++ {
		var sum int = 0

		for j := 0; j < len(accounts[i]); j++ {
			sum += accounts[i][j]
		}

		if res < sum {
			res = sum
		}
	}

	return res
}

func main() {
	arr := [][]int{
		{1, 2, 3},
		{2, 3, 4},
	}
	fmt.Println(maximumWealth(arr))
}

File: 1672/c++/main.cpp
#include <iostream>
#include <vector>

using namespace std;

int maximumWealth(vector<vector<int>> &accounts)
{
    int res = 0;

    for (int i = 0; i < accounts.size(); i++)
    {
        int sum = 0;
        for (int j = 0; j < accounts[i].size(); j++)
        {
            sum += accounts[i][j];
        }
        res = max(res, sum);
    }

    return res;
}

int main()
{
    vector<vector<int>> input{
        {1, 2, 3},
        {2, 3, 4},
    };

    cout << maximumWealth(input) << endl;
};